library/src/containers/AsyncApi/AsyncApi.tsx

```diff
@@ -53,12 +60,28 @@ class AsyncApiComponent extends Component<AsyncApiProps, AsyncApiState> {
   }
 
   async componentWillMount() {
```

Comment:
Heads up `componentWillMount` is deprecated. Check out: https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html.